crates/diesel_models/src/query/payment_attempt.rs

```diff
@@ -273,4 +276,32 @@ impl PaymentAttempt {
 
         Ok(filters)
     }
+    pub async fn get_total_count_of_attempts(
+        conn: &PgPooledConn,
+        merchant_id: &str,
+        constraints: &PaymentListFilterConstraints,
```

Comment:
Instead of trying to use a struct `PaymentListFilterConstraints` you can accept the individual fields as input preventing the above struct re-declarations